src/assets/data/markdown/ionic-react/unit-tests.md

```diff
@@ -2,34 +2,55 @@
 
 In this lab, you will learn how to:
 
-- Run the existing suite of unit tests
+- Setup support for CI, code coverage, and snapshots
+- Create and use mock objects
 - Structure Unit Tests
-- Add component unit tests
 
-## Run the Tests
+## Setup CI, Coverage, and Snapshot Support
```

Comment:
I left it out on the existing React training, but I wonder if we shouldn't include a link to "@testing-library/react" now that we include it with the starter project.

It used to be that I had students install that and re-work the test to use it. But then on the latest go-around I noticed that the FW team already had done that in the starter at some point.